news/12769.bugfix.rst

```diff
@@ -0,0 +1 @@
+Gracefully handle Windows registry access errors while guessing the MIME type of a file.
```

Comment:
Note for other reviewers, while the code comment is more general because it seems guess_type() can fail on any platform (according to the docs), the only reports in practice are on Windows w/ the registry.